src-docs/src/views/page/page_template_example.js

```diff
@@ -0,0 +1,759 @@
+              retaining any responsive behavior and allowing normal page scroll
+              on smaller screens.
+            </li>
+          </ol>
+          <EuiCallOut
+            color="warning"
+            iconType="accessibility"
+            size="s"
+            title={
+              <>
+                When applying the <EuiCode>.eui-yScroll</EuiCode> class, it is
+                recommended to also apply <EuiCode>tabindex=0</EuiCode> to
+                ensure keyboard users can scroll these containers.
+              </>
+            }
+          />
+        </>
+      ),
+      demo: (
+        <PageDemo
+          slug="full-height"
+          pattern={PageFullHeight}
+          template={PageFullHeightTemplate}
+        />
+      ),
+      fullScreen: {
+        slug: 'full-height',
+        demo: (
+          <PageDemo
+            slug="full-height"
+            pattern={PageFullHeight}
+            template={PageFullHeightTemplate}
+            fullscreen
+          />
+        ),
+      },
+    },
+    {
+      title: 'Simple layout with centered body',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: PageSimpleCenteredBodyTemplateSource,
+          displayName: 'Template JS',
+        },
+        {
+          type: GuideSectionTypes.JS,
+          code: PageSimpleCenteredBodySource,
+          displayName: 'Components JS',
+        },
+      ],
+      text: (
+        <p>
+          Similar to the version with a side bar, when the content for the page
+          is minimal or in an empty/pre-setup state, the page content can be
+          centered vertically and horizontally. We recommend then using the{' '}
+          <Link to="/display/empty-prompt">
+            <strong>EuiEmptyPrompt</strong>
+          </Link>{' '}
+          for the content.
+        </p>
+      ),
+      demo: (
+        <PageDemo
+          slug="simple-centered-body"
+          centered
+          pattern={PageSimpleCenteredBody}
+          template={PageSimpleCenteredBodyTemplate}
+        />
+      ),
+      fullScreen: {
+        slug: 'simple-centered-body',
+        demo: (
+          <PageDemo
+            slug="simple-centered-body"
+            centered
+            pattern={PageSimpleCenteredBody}
+            template={PageSimpleCenteredBodyTemplate}
+            fullscreen
+          />
+        ),
+      },
+    },
+    {
+      title: 'Simple layout with centered content',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: PageSimpleEmptyContentTemplateSource,
+          displayName: 'Template JS',
+        },
+        {
+          type: GuideSectionTypes.JS,
+          code: PageSimpleEmptyContentSource,
+          displayName: 'Components JS',
+        },
+      ],
+      text: (
+        <p>
+          Also similar to the previous examples, you can create a centered panel
+          to emphasis incompleteness even with a page header. For this setup,
```

Comment:
I'm not sure... But... 

```suggestion
          to emphasize incompleteness even with a page header. For this setup,
```